src/main/resources/init_db.sql

```diff
@@ -7,3 +7,10 @@ CREATE TABLE `manufacturers` (
                                         `country` VARCHAR(225) NOT NULL,
                                         `is_deleted` TINYINT NOT NULL DEFAULT 0,
                                         PRIMARY KEY (`id`));
+CREATE TABLE `drivers` (
+                           `id` bigint NOT NULL AUTO_INCREMENT,
+                           `name` varchar(225) DEFAULT NULL,
+                           `licenseNumber` varchar(225) DEFAULT NULL,
```

Comment:
It's better not to mix different styles for naming fields in tables. If you use underscore ('is_deleted') - use it also for other fields.
```suggestion
                           `license_number` varchar(225) DEFAULT NULL,
```